### DISCUSSION TITLE
"latest" image

### DISCUSSION THREAD

**Discussion by alexbn71** [doitandbedone/ispyagentdvr-docker#482] [General]
Just to understand ... I understood that the creation of the "last" image was automatic while although the tag 4.2.3.0 is already there, the "last" is still 4.2.0.0

Thanks
Alex

**Comment by doitandbedone** [comment]
This is a question, not a bug. Please use discussions for quick questions or doubts. 

Builds are scanned every 4 hours.  If a new version is available then a PR will be created automatically and a tag build and image published. You may always point manually to the tag by adding :<your-desired-tag>. For instance: doitandbedone/ispyagentdvr:4.2.3.0. Tags are built immediately and published within minutes. So you don't have to wait for default to catch up.

**Reply by alexbn71** [reply]
Sorry, thank you

**Comment by alexbn71** [comment]
On a system that automatically pull the "latest" tag when detect a change it's not obliously a solution keep tracks of tags of newer version. 

Does is "latest" manually generated by you? because is still two version back of the current.

**Comment by alexbn71** [comment]
latest version is never aligned and stay back every time for days :-(
